tests/kola/version/toolbox

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This is RHCOS only, we ship a different version of toolbox in FCOS
+
+# kola: { "exclusive": false }
+set -xeuo pipefail
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+if [ "$(file -b /usr/bin/toolbox)" != "Bourne-Again shell script, ASCII text executable" ]; then
```

Comment:
This is fine though if you'd like, another approach that gets to the heart of it is to look at the URL, e.g.:

```sh
if [ "$(rpm -q --qf '%{URL}' toolbox)" != "https://github.com/coreos/toolbox" ]; then
```

Reply:
Ooh, I like this better.  Thanks Jonathan!